Sem. 03/10.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+#include <cmath>
+using namespace std;
+int main()
+{
+    int number,count=0;
+    cin>>number;
+    int n=number;
+    while(n>0){
+        count++;
+        n/=10;
+    }
+    
+    if(count%2==0){
+        for(int i=1;i<=count/2;i++){
```

Comment:
In for-loops, as on lines 15 and 22, it is not ok to do calculations in the *condition* field, because this way, on every iteration, you would be doing the same calculation over and over again. Just store the calculation in some local variable and use it, instead of doing the calculation on every iteration.